src/test/java/it/gov/pagopa/spontaneouspayment/service/ServicesServiceTest.java

```diff
@@ -204,4 +206,97 @@ void getServiceDetails_404() {
 			fail();
 		}
 	}
+	
+	@Test
+	void createService() {
+		assertTrue(emulator.isRunning());
+		ServiceProperty sp5 = new ServiceProperty("propName5", PropertyType.STRING, true);
+		Service s5 = new Service();
```

Comment:
if you have time, create a method in TestUtil to generate this mock
For example:
`
Service getMockService(String seed);
`